config/routes.rb

```diff
@@ -202,6 +203,20 @@
 
   root 'v0/example#index', module: 'v0'
 
+  scope '/services' do
+    namespace :v0, defaults: { format: 'json' } do
+      namespace :docs, only: [] do
+        # namespace :health do
+        #   resources :prescriptions
+        #   resources :secure_messages
+        # end
+        # resources :health, only: [:index]
```

Comment:
what are these comments for?

Reply:
Considering breaking out vets-api endpoints into multiple services currently associated with tags to provide a better breakdown of models and more context for developers navigating the documentation.